groundstation/static/js/components/CommunicationsListFull.jsx

```diff
@@ -34,22 +68,36 @@ const CommunicationEntry = (props) => {
     }
     infoTimestamp = new Date(infoTimestamp);
     let infoStringTimestamp = formatDateToUTCString(infoTimestamp);
-
+    
     return (
         <div style={divStyle}>
             <div>
                 <p style={infoStringStyle}>{infoStringPrefix} <strong>{sender}</strong> on: </p><p>{infoStringTimestamp}</p>
             </div>
             <p style={contentStyle}>Message: {props.entry.message}</p>
-            <p style={contentStyle}>Is Queued: {props.entry.is_queued ? 'true' : 'false'}</p>
+            {(sender === 'comm') ? null: <p style={contentStyle}>Is Queued: {props.entry.is_queued ? 'true' : 'false'}</p>}
+            <div>
+            {((props.is_admin || (sender === username)) && (sender !== 'comm') && (showQueueButton)) ? <Button 
+                style={{marginBottom:"1%"}} 
+                color = "primary" 
+                onClick={queueHandler}
+                variant = 'outlined'>{props.entry.is_queued ? 'De-Queue': 'Queue'}</Button>: null}
+            </div>
         </div>
     );
 }
 
 
 const CommunicationsList = (props) => {
+    const [isAdmin, setIsAdmin] = useState(false);
+    //const[showQueueButton, setQueueButton] = useState(false);
     const divStyle = {margin:'2%'}
 
+    // if (props.showQueueButton === true) {
+    //     setQueueButton(props.showQueueButton);
+    // }
+    console.log(props.showQueueButton);
```

Comment:
```suggestion
    const divStyle = {margin:'2%'}
```
Remove debug print and commented-out code